lib/visitor.js

```diff
@@ -589,17 +593,19 @@ class Visitor {
      * @param {EntityCSN} element - CSN data belonging to the the element.
      * @param {SourceFile} file - the namespace file the surrounding entity is being printed into.
      * @param {Buffer} [buffer] - buffer to add the definition to. If no buffer is passed, the passed file's class buffer is used instead.
+     * @param {TypeResolveOptions} [resolverOptions] - custom type resolver options
      * @returns @see InlineDeclarationResolver.visitElement
      */
-    visitElement(name, element, file, buffer = file.classes) {
+    visitElement(name, element, file, buffer = file.classes, resolverOptions) {
```

Comment:
This feels a bit awry, as it introduces an optional parameter without explicit default value after ones with explicit default values (sonarlint complains about this as well).
I think this should be streamlined with [`InlineDeclarationResolver::visitElement`](https://github.com/cap-js/cds-typer/blob/main/lib/components/inline.js#L78) where we pass a destructured object instead. That was due before your PR, but with the introduction of yet another parameter, this feels even more awkward. Could you please incorporate this change?

Reply:
Agreed. I made the adjustment